packages/react-sdk/src/hooks/useFlowScheduleManager.ts

```diff
@@ -0,0 +1,442 @@
+import {arg, t} from "@onflow/fcl"
+import {UseQueryOptions} from "@tanstack/react-query"
+import {useCallback, useMemo} from "react"
+import {useFlowQueryClient} from "../provider/FlowQueryClient"
+import {useFlowChainId} from "./useFlowChainId"
+import {useFlowClient} from "./useFlowClient"
+import {useFlowMutate} from "./useFlowMutate"
+import {useFlowQuery} from "./useFlowQuery"
+
+export enum TransactionPriority {
+  High = 0,
+  Medium = 1,
+  Low = 2,
+}
+
+export enum TransactionStatus {
+  Unknown = 0,
+  Scheduled = 1,
+  Executed = 2,
+  Canceled = 3,
+}
+
+export interface TransactionData {
+  id: string
+  priority: TransactionPriority
+  executionEffort: string
+  status: TransactionStatus
+  fees: string
+  scheduledTimestamp: string
+  handlerTypeIdentifier: string
+  handlerAddress: string
+  // Optional handler data when includeTransactionHandler is true
+  handler?: HandlerData
+}
+
+export interface HandlerData {
+  typeIdentifier: string
+  // Not yet a standard so we need to let the client decide the type
+  metadata: {[viewType: string]: any}
+  // Optional transactions when includeHandlerTransactions is true
+  transactions?: TransactionData[]
+}
+
+export interface HandlerCollection {
+  [typeIdentifier: string]: HandlerData
+}
+
+export interface ScheduleManagerFilter {
+  // Include transactions for each handler
+  handlerTransactions?: boolean
+  // Include handler for each transaction
+  transactionHandler?: boolean
+}
+
+export interface UseFlowScheduleManagerArgs {
+  // Manager address
+  address?: string
+  // Filter options for including related data
+  include?: ScheduleManagerFilter
+  // React Query settings
+  query?: Omit<UseQueryOptions<unknown, Error>, "queryKey" | "queryFn">
+  flowClient?: ReturnType<typeof useFlowClient>
+}
+
+export interface UseFlowScheduleManagerData {
+  transactions: TransactionData[]
+  handlers: HandlerCollection
+}
+
+export interface UseFlowScheduleManagerResult {
+  data: UseFlowScheduleManagerData | null
+  isLoading: boolean
+  isFetching: boolean
+  error: Error | null
+  refetch: () => void
+
+  // Query scripts
+  scripts: {
+    getManagerTransactions: (
```

Comment:
This feels a little repetitive. 

```
const manager = useFlowScheduleManager({ address })

await manager.scripts.getManagerTransactions(address)
```

You can probably leave out "Manager" from the function name. What do you think?